packages/browser/src/core/user/index.ts

```diff
@@ -398,3 +398,53 @@ export class Group extends User {
     return undefined
   }
 }
+
+export class UniversalStorage {
+  private cookies: Store
+  private localStorage: Store
+  private mem: Store
+
+  options: UserOptions = {}
```

Comment:
what does this do here?

Reply:
Hah, good catch, has no use ( remains of copy 🍝 )